### ISSUE TITLE
not complete user list

### ISSUE THREAD

**Issue by akitzing** [wekan/wekan#5020]
* Did you test in newest Wekan?: yes
* Did you configure root-url correctly so Wekan cards open correctly (see https://github.com/wekan/wekan/wiki/Settings)? yes
* Operating System: Suse
* Deployment Method (Snap/Docker/Sandstorm/bundle/source): Docker (Swarm)
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): Traefik
* Node.js Version: The one in Docker Image
* MongoDB Version: 4 and 6
* What webbrowser version are you using (Wekan should work on all modern browsers that support Javascript)? Chrome

### Problem description
The user list is not complete. For example, 25/26 users are displayed, but there are many more. Unlisted users can only be found via the search function. We have a ldap-connection.
We already had this problem with previous versions.
Many thanks

**Comment by xet7** [comment]
@akitzing 

Yes, it only shows amount of users loaded to browserside minimongo. Not all users are loaded there currently.

It has not been fixed yet to show real count of users at serverside MongoDB database.

I think there is some WeKan REST API to get real count, but it is not yet used in Admin Panel Web UI.

**Comment by vasyugan** [comment]
> @akitzing
> 
> Yes, it only shows amount of users loaded to browserside minimongo. Not all users are loaded there currently.
> 
> It has not been fixed yet to show real count of users at serverside MongoDB database.
> 
> I think there is some WeKan REST API to get real count, but it is not yet used in Admin Panel Web UI.

Is there a way to force display of the full list? This is quite irritating. At times, the list displays only 26 users, at other times I get the full list. 
Also the display of the users' attributes seem to be erratic. Sometimes, some users suddenly have no email address and/or no associated trello user, at other times they do have them.

**Comment by xet7** [comment]
@vasyugan 

> Is there a way to force display of the full list?

Not yet at Admin Panel. PRs welcome.

You can use API to list users:
https://github.com/wekan/wekan/blob/main/api.py

**Comment by vasyugan** [comment]
> @vasyugan
> 
> > Is there a way to force display of the full list?
> 
> Not yet at Admin Panel. 

What surprises me is that choosing a different browser or emptying the cache does not make a difference. If, as you say, this minimongo thing is a browserside application, shouldn't this be different?

> PRs welcome.

I'm not a programmer. So unfortunately I've got nothing to contributes, sorry!

> You can use API to list users: https://github.com/wekan/wekan/blob/main/api.py

Thanks. I'll raise this possibility in our group.

**Comment by xet7** [comment]
@vasyugan

> I'm not a programmer. So unfortunately I've got nothing to contributes, sorry!

It's also possible to sponsor developing features:
https://wekan.team/commercial-support/

**Comment by xet7** [comment]
This is already fixed
